internal/e2e/go.mod

```diff
@@ -0,0 +1,69 @@
+module github.com/solarwinds/solarwinds-otel-collector/internal/e2e
+
+go 1.23.4
+
+require (
+	github.com/stretchr/testify v1.10.0
+	github.com/testcontainers/testcontainers-go v0.34.0
+	go.opentelemetry.io/collector/pdata v1.21.0
```

Comment:
The version of `go.opentelemetry.io/collector/pdata` is not in sync with other modules. It should be `v1.19.0`.

Reply:
indeed, updated